cs/traits.go

```diff
@@ -282,7 +282,7 @@ func caSpec() PRTSpec {
 
 	spec.Instaforming = true
 	spec.PermaformChance = .1 // chance is 10% if pop is over 100k
-	spec.PermaformPopulation = 100_000
+	spec.PermaformPopulation = 100000
```

Comment:
Can you revert this please? golang supports adding thousands separators to numeric literals.